### ISSUE TITLE
"The software was closed because an error occurred." on installed games.

### ISSUE THREAD

**Issue by JimmyCarrero** [Atmosphere-NX/Atmosphere#621]
I've been having issues when getting games to boot. Atmosphere itself boots up perfectly well, be it through Hekate or fusee primary. The issue come into play when i try to boot into an installed game. whenever i do, it shows the "The software was closed because an error occurred." message. 

It only happens with nsp-based installs. The physical cart games i have load perfectly, even if they have nsp dlc installed. (e.g: I have Zelda botw in a physical cartridge, and I installed the latest updates and dlc through unofficial means. When i try to play this, it works perfectly well. DLC and all).

The problem is only with installed games. I've made sure the nsp files are clean and verified with SXOS, in which all the games boot and play perfectly. I've formatted my SD card to Fat32 various times. Tried various custom atmosphere packages (Kosmos, Stardust) and the official releases (Hekate, Atmosphere) but to no avail. In hekate i've messed with the .ini files to see if it made any difference.

In Summary: 
In atmosphere CFW (booted through either fusee-primary or hekate) none of my installed games boot. They get stuck on the "the software was closed because an error occurred" message. This does not happen to cartridge games even when they have DLC and updates that have been installed through unofficial means. Now, when the exact same setup and games are used in SXOS, they work perfectly fine. I've tried everything that comes to mind and reinstalled the software various times, from different official and unofficial sources (Kosmos, Stardust, SD-setup.nro) to no avail. Below are my switch specifications should they be any help.

My Switch specifications:
Firmware: 8.1.0
SD card: 128GB
Serial Number: XAW40005831292

**Comment by fincs** [comment]
> nsp-based installs

> The problem is only with installed games. I've made sure the nsp files are clean and verified with SXOS,

We don't support piracy.
